src/Components/Forms/OrderForm/Forms/ShippingForm.tsx

```diff
@@ -0,0 +1,128 @@
+import { Field } from "../../support"
+import InvertedButton from "../../../Buttons/Inverted"
+import React, { Fragment } from "react"
+import Text from "../../../Text"
+import Title from "../../../Title"
+import colors from "../../../../Assets/Colors"
+import { Grid, Row, Col } from "react-styled-flexboxgrid"
+import { WizardStepChildProps } from "../../../Wizard/types"
+
+export const ShippingForm: React.SFC<WizardStepChildProps> = ({
+  form,
+  wizard,
+}) => {
+  return (
+    <Grid fluid>
+      <Row>
+        <Col xs>
+          <Title titleSize="medium">Shipping details</Title>
+        </Col>
+      </Row>
+      <Row>
+        <Col xs>
+          <Text textSize="medium">
+            Upon processing your order, an Artsy Specialist will connect you
+            with the seller for shipping arrangements. Any shipping fees will be
+            collected at that time.
+          </Text>
+        </Col>
+      </Row>
+      <Row>
+        <Col xs>
+          <Title titleSize="small">Shipping address</Title>
+        </Col>
+      </Row>
+
+      <AddressFormInputs />
+
+      <Row>
+        <Col xs>
+          <InvertedButton block onClick={form.handleSubmit}>
```

Comment:
Still wondering what the best practice is for adding logic in the `submit` function on each step, since it seems that `handleSubmit` is defined on the parent. Would you just add a function that does an action and then calls `handleSubmit` explicitly?

I'm also happy to look into this when I pick up the next task, just wondering if there's already a preferred pattern.

Reply:
Yeah, its hard to explain but:
`<Wizard>`'s children get `wizard` and `form` props which contain the same props as the Wizard `Step`'s child. In the latter case those props come down from context, which seems to cause some typing confusion. Ultimately I see it as- if you want a simple control in your layout (the `Wizard` child) then you can use it there. If you would rather have it in the `Step` (eg, "continue to step 2") you can use the same props there (but under the hood they come from context via the wizard)